cmd/example-hook-sidecar/smbios.go

```diff
@@ -39,7 +39,10 @@ import (
 	domainSchema "kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 )
 
-const baseBoardManufacturerAnnotation = "smbios.vm.kubevirt.io/baseBoardManufacturer"
+const (
+	baseBoardManufacturerAnnotation         = "smbios.vm.kubevirt.io/baseBoardManufacturer"
```

Comment:
This is not used.

Reply:
What do you mean? what is not being used?
"baseBoardManufacturerAnnotation" is being used in this file

Reply:
Ahh you are right. I thought it was added as new one. Sorry.